src/FakeItEasy/Core/WrappedObjectRule.cs

```diff
@@ -50,8 +52,16 @@ public void Apply(IInterceptedFakeObjectCall fakeObjectCall)
             Guard.AgainstNull(fakeObjectCall, nameof(fakeObjectCall));
 
             var parameters = fakeObjectCall.Arguments.GetUnderlyingArgumentsArray();
-            var valueFromWrappedInstance = fakeObjectCall.Method.Invoke(this.wrappedObject, parameters);
-            fakeObjectCall.SetReturnValue(valueFromWrappedInstance);
+            try
+            {
+                var valueFromWrappedInstance = fakeObjectCall.Method.Invoke(this.wrappedObject, parameters);
+                fakeObjectCall.SetReturnValue(valueFromWrappedInstance);

```

Comment:
Good question. I don't think so, but it couldn't hurt to move it outside the `try`. I'll do it